data/sea/20-simple.h

```diff
@@ -31,6 +31,7 @@ static idouble_t INLINE iint_extend   (iint_t    x)              { return x; }
 static iint_t    INLINE iint_neg      (iint_t    x)              { return -x; }
 static iint_t    INLINE iint_add      (iint_t    x, iint_t    y) { return x + y; }
 static iint_t    INLINE iint_sub      (iint_t    x, iint_t    y) { return x - y; }
+static iint_t    INLINE iint_mod      (iint_t    x, iint_t    y) { return x % y; }
```

Comment:
looks like you did add modulus?

Reply:
only to the backend yet. heh.

Reply:
what's the difference in semantics between the integer and double variants?